noxfile.py

```diff
@@ -20,7 +20,7 @@
 PACKAGE = str("lib" / Path("iris"))
 
 #: Cirrus-CI environment variable hook.
-PY_VER = os.environ.get("PY_VER", ["3.6", "3.7", "3.8"])
+PY_VER = os.environ.get("PY_VER", ["3.7", "3.8"])
```

Comment:
Could /should this use the same discovery code that you've written for the docs?

Reply:
What we're actually testing in nox is not a specific version of python, but a specific environment that is defined by a yaml file in `requirements/ci`.

Reply:
Hmmm I'll give it a try and see what it looks like... the hardest part is deciding on where such common code will live.

But yeah, that would be a nice win :+1:

Reply:
@bjlittle I'm happy for this to go in as-is with your latest updates.  Let me know if you plan to rework otherwise I'm merging today

Reply:
@jamesp Okay, so I gave this a reasonably punt... but you swiftly run into a bit of a pickle as everything under the `requirements` directory isn't installed as a package, so you can't really safely reference it from `nox` or the docs without jumping through some hoops.

It is possible, and I suspect we'd either want to re-structure our resources or use `setuptools` sub-packages to do that properly... but I'm not really inclined to pursue this ATM, unless you want me to follow through on this?

Reply:
I'm surprised this is an issue. Sure, we run our tests on a installed instance of Iris, so there are concerns on inclusion/exclusion there. But as far as I was aware Nox was expected to operate on a checkout of a repo, rather than within an install. Indeed I've based a few things on this assumption.

Reply:
I mean, you must be referencing `requirements` _somewhere_ from `nox` as those environments yamls are being read to create the test envs. But I think I understand your point to be that the introspection bit of nox where it works out what sessions it has might not be able to safely operate.  either way though, nox isn't going to work if the `requirements/ci` folder isn't there.

This isn't a big issue, happy to leave it hard coded.

Reply:
Just to be clear... the issues isn't about whether the directory exists, per se... rather that I was attempting to bank common code under `requirements/__init__.py` for both the docs and `nox` to use - that's the problem here.

The simple alternative is to put common code within `iris` (which I really don't want to do) or I duplicate the code within the `noxfile` and within the docs.

So it's do-able, just whatever you prefer... if you want to follow-thru with this

Reply:
Either way, I don't think this should be a blocker for this PR. We can happily bank this PR "as is", and iterate on this later.

Reply:
Also, for me this is all becoming a bit of a moot point, as a developer still requires to manually update the `matrix` for the minimal and full tests.